apix-impl/build.gradle

```diff
@@ -33,23 +37,16 @@ allprojects {
     apply plugin: 'java-library'
 
     dependencies {
-        api(project(":apix-interface")) {
-            exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
-            exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
```

Comment:
We don't have to exclude these dependencies because on apix-interface they are add as "compileOnly"